synapse/events/utils.py

```diff
@@ -506,18 +512,16 @@ def _inject_bundled_aggregations(
             }
 
         # If this event is the start of a thread, include a summary of the replies.
-        if aggregations.thread:
-            thread = aggregations.thread
+        if event_aggregations.thread:
+            thread = event_aggregations.thread
 
             # Don't bundle aggregations as this could recurse forever.
```

Comment:
this comment seems to be misplaced now. And do we need to worry about it recursing forever?

Reply:
I think we do still in some odd circumstances. I need to ponder this a little more.